src/translations/nbTranslations.json

```diff
@@ -1488,8 +1488,10 @@
         "if_same_press_no_in_box": "Om dette er samme resultat som din registrering, må du trykke 'NEI' i denne boksen for å unngå å publisere.",
         "publication_already_exists": "Det finnes allerede en publikasjon med samme tittel, år og kategori som din registrering."
       },
-      "error_description": "Følgende feil må rettes opp før publisering av resultat:",
```

Comment:
NB! Fordel for oversetter om vi husker å duplisere disse til `_workflow1` i lokalise rett før merge, sånn at de slipper å legge inn samme oversettelser somligger der i dag på engelsk og bokmål